docs/user_authentication/third_party_token_management.md

```diff
@@ -0,0 +1,25 @@
+
+```sequence {theme: 'simple'}
+Title: Third party idp token management
+Pbench-Dashboard->Pbench-Server:[1] GET login page
+Pbench-Server->Pbench-Dashboard:[2] 200 login page
+Pbench-Dashboard->Pbench-Dashboard:[3] Generate Authorization request
+Pbench-Dashboard->Identity-Provider:[4] POST Authorization request
+Identity-Provider->Identity-Provider:[5] 200 Authenticate User
+Identity-Provider->Identity-Provider:[6] GET consent
```

Comment:
The `Pbench-dashboard` won't be asking for consent from the user directly. It's the identity provider who takes the consent from the user for the Pbench client about what information it is about to share with Pbench. So getting user consent solely happens on the identity-providers site.

Reply:
I should probably get rid of `GET consent` step since it's causing confusion and also it is a part of the identity provider's flow of authenticating users and not something we should worry about how it happens.

Reply:
> I should probably get rid of GET consent step since it's causing confusion and also it is a part of the identity provider's flow of authenticating users and not something we should worry about how it happens.

😀  I suggested that last Thursday, but was shot down.  Peter wants the full set of state transitions, but I'm not seeing how we can represent that without splitting the Dashboard from the Browser, because, for several of the transitions, I'm pretty sure that the Identity-provider is taking over the Browser from the Dashboard....

Reply:
> getting user consent solely happens on the identity-providers site.

I think we need to be more specific that that:  I believe that consent is obtained by the user interacting with an Identity-provider page in the Browser which makes an appropriate request of the Identity-provider service.